asset_dashboard/static/js/components/table_utils/projectColumns.js

```diff
@@ -22,69 +22,75 @@ const projectColumns = (selector) => {
       accessor: 'name'
     },
     {
-      Header: 'Description',
-      accessor: 'description', 
+      Header: 'Estimated Cost',
+      accessor: 'budget',
+      Cell: props => props.value.toLocaleString() // adds commas
```

Comment:
Yeah that's a feature of the [react-table package](https://www.npmjs.com/package/react-table).

It looks like the maintainer of that library did a major rebrand and now the docs are very confusing, but I think this might be it? https://tanstack.com/table/v8/docs/api/core/cell

Otherwise there are a ton of stackoverflows that can show you how you can use that, like: https://stackoverflow.com/questions/45968791/react-table-custom-cell-component-that-references-several-row-properties